### PR TITLE
Scroll integration 1

### PR DESCRIPTION
**Pull request by alhonaut**

### CODE REVIEW

File: src/app/join-client/page.tsx [hunk 1]
```diff
@@ -67,84 +67,40 @@ const JoinClient = () => {
...
-              />
-            </div>
-            <div className={styles.nameInputFields1}>
-              <b className={styles.yourCompanyName}>Your company Name</b>
-              <input
-                className={styles.textInput1}
-                placeholder="e.g Roaring Production"
-                type="text"
-              />
+    <div id="join-freelance" className={styles.modal}>
```
Comment (mgroovyank) [inline src/app/join-client/page.tsx:70]: @alhonaut I had already worked on join client and join freelancer.

File: src/app/components/join/join-client.module.css [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: rgb(214, 234, 94, 0.4);
-  z-index: 100000;
-  transition: ease-out 0.3s;
-}
+.container23 {
```
Comment (mgroovyank) [inline src/app/components/join/join-client.module.css:1]: I'm using modal backdrop class

Comment (mgroovyank) [reply src/app/components/join/join-client.module.css:1]: Not sure why you made this change in join client?

File: src/app/components/join/join-client.module.css [hunk 3]
```diff
@@ -24,20 +27,18 @@
...
   width: 48px;
   height: 48px;
   position: relative;
 }
-.addPhotoUser {
-  width: 154px;
-  height: 154px;
-  position: relative;
-}
+
```
Comment (mgroovyank) [inline src/app/components/join/join-client.module.css:36]: I need this class as well

File: src/app/components/join/join-client.module.css [hunk 4]
```diff
@@ -289,29 +321,32 @@
   font-size: var(--font-size-13xl);
   color: var(--light);
   font-family: var(--font-unbounded);
-  margin: 0 auto;
```
Comment (mgroovyank) [inline src/app/components/join/join-client.module.css:292]: I need this as well

File: src/app/components/join/join-client.modules.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
+/*GLOBAL STYLING*/
```
Comment (mgroovyank) [inline src/app/components/join/join-client.modules.css:1]: this file is not required. Should be deleted

File: src/app/layout.tsx [hunk 6]
```diff
@@ -1,22 +1,12 @@
 import { Fragment } from "react";
-import type { AppProps } from "next/app";
 import "./styles/global.css";
```
Comment (mgroovyank) [inline src/app/layout.tsx:2]: why these changes are made?

File: src/middleware.ts [hunk 7]
```diff
@@ -12,13 +12,15 @@ export async function middleware(request: NextRequest) {
   const isFreelancerDashboard = pathname.startsWith("/freelancer-dashboard");
   const isChatWindow = pathname.startsWith("/chat");
   const isAuth = await isLoggedIn();
```
Comment (mgroovyank) [inline src/middleware.ts:14]: Double changes here

**Review by mgroovyank — APPROVED** [review]

**Review by mgroovyank — APPROVED** [review]

**Review by Horlarmmy — APPROVED** [review]
